src/common/paddle/index.ts

```diff
@@ -9,14 +9,14 @@ import {
   type TransactionReadyEvent,
   type TransactionUpdatedEvent,
 } from '@paddle/paddle-node-sdk';
-import { logger } from '../logger';
+import { logger } from '../../logger';
```

Comment:
Just moved this file to a folder. I feel like this file will be super cluttered if I put the newly introduced block of codes here.

Reply:
Maybe it can still be called `paddle.ts`?